### PR TITLE
Use file scoped namespaces

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
Best viewed at https://github.com/dotnet/aspnetcore/pull/38076/files?w=1. I ran the format tool and it did all of the work here.

**Comment by Tratcher** [comment]
Looks like this will reset/hide git blame history on every file. Blame doesn't exclude whitespace changes.
https://github.com/dotnet/aspnetcore/blame/bac555c2e6a1a6f9f5fd839918b4afca7e5c6c6d/eng/tools/RepoTasks/CreateFrameworkListFile.cs

**Comment by pranavkm** [comment]
Animation

^ Works no?

**Comment by Tratcher** [comment]
Oh it still works, but you've touched most lines so a lot of changes will be hidden behind this one.

**Comment by Tratcher** [comment]
This does not include most samples or tests, correct? I think we'd want those to be consistent as well.

**Comment by pranavkm** [comment]
> This does not include most samples or tests, correct?

Just doing it piecemeal so as to not overwhelm the review. My plan is to make it a build warning if you don't use the feature. #bring-back-stylecop.

**Review by Tratcher — APPROVED** [review]
Consider us overwhelmed 😁. I spot checked things, but could not review every file.

**Comment by dougbu** [comment]
Actually, best viewed in CodeFlow. Can add the CodeFlow Chrome extension (<[http://codeflow/content/workflows-github-pullrequest.html#OpenPRChrome](http://codeflow/content/workflows-github-pullrequest.html#OpenPRChrome)>) to either Chrome or Edge.

---
In reply to: [960186866](https://github.com/dotnet/aspnetcore/pull/38076#issuecomment-960186866) [](http://example.com/codeflow?ancestors=960186866,960186866)

**Comment by dougbu** [comment]
I'm a bit more concerned about the future inability to backport almost _any_ change from main.

---
In reply to: [960236457](https://github.com/dotnet/aspnetcore/pull/38076#issuecomment-960236457) [](http://example.com/codeflow?ancestors=960236457,960236457)

File: src/Hosting/Server.Abstractions/src/ServerIntegratedAuth.cs [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-    /// Used by servers to advertise if they support integrated Windows authentication, if it's enabled, and it's scheme.
+    /// Indicates if integrated Windows authentication is enabled for the current application instance.
     /// </summary>
-    public class ServerIntegratedAuth : IServerIntegratedAuth
-    {
-        /// <summary>
-        /// Indicates if integrated Windows authentication is enabled for the current application instance.
-        /// </summary>
-        public bool IsEnabled { get; set;}
+    public bool IsEnabled { get; set; }
```
Comment (dougbu) [inline src/Hosting/Server.Abstractions/src/ServerIntegratedAuth.cs:14]: I just _know_ this was a test to see who would catch you slipping the space in here :smile:

Comment (dougbu) [reply src/Hosting/Server.Abstractions/src/ServerIntegratedAuth.cs:14]: Found a half dozen or so new spaces. Should we be looking for more❔

**Comment by dougbu** [comment]
This is a huge change even if 1480 is a small percentage of the files in the repo. Do we really want to make 'main' completely inconsistent with our servicing branches❔

**Comment by pranavkm** [comment]
>  Do we really want to make 'main' completely inconsistent with our servicing branches❔

I'd be happy to bring this up in CI council. I know individual files have been changed to use this formatting as people have touched them, so we definitely seem to like / want it, but I'll let the team form a consensus on this.

**Comment by dougbu** [comment]
> I'd be happy to bring this up in CI council.

CI Council is a larger group. Suggest we put this on the Dept. of Infrastructure agenda instead. Hmm, I remain unsure if that group will meet tomorrow (@javiercn ❔)

**Comment by pranavkm** [comment]
🆙 📅 to include all the files and enables the build warning.

**Comment by pranavkm** [comment]
I'll go ahead and set this to auto-merged to avoid the ongoing merge conflicts. We can have follow ups to review / adjust the code.

**Pull request merged**